src/components/access_group/AccessGroupHistoryEdit.vue

```diff
@@ -0,0 +1,115 @@
+<template>
```

Comment:
Got remove from designs afaik.

Reply:
For now. I thought it might come back though in future versions.